src/main/java/racingcar/RacingGame.java

```diff
@@ -0,0 +1,63 @@
+package racingcar;
+
+import racingcar.domain.RacingCars;
+import racingcar.domain.Winners;
+import racingcar.domain.moving.DefaultMovingStrategy;
+import racingcar.domain.moving.MovingStrategy;
+import racingcar.domain.RacingCar;
+import racingcar.domain.moving.Position;
+import racingcar.domain.moving.RandomValueProvider;
+import racingcar.view.input.InputView;
+import racingcar.view.result.ResultView;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RacingGame {
+
+    private final InputView inputView;
+    private final ResultView resultView;
+
+    private final MovingStrategy movingStrategy;
+    private List<RacingCar> racingCars;
```

Comment:
일급 컬렉션인 `RacingCars`를 사용해 보세요.

```suggestion
    private RacingCars racingCars;
```